config/routes.rb

```diff
@@ -9,6 +9,7 @@
     namespace :v1 do
       resources :upload, only: [:create], controller: 'search_results'
       resources :search_results, only: [:index, :show]
+      resources :export, only: [:index], controller: 'exports'
```

Comment:
The controller should be recognized by default, no?
And as this is to export a search result, could we use a [nested route](https://guides.rubyonrails.org/routing.html#nested-resources) for it?
```suggestion
      resources :search_results, only: [:index, :show] do
      	resource :export, only: [:show]
      end
```